src/RadixDlt.NetworkGateway.GatewayApi/Exceptions/InternalServerException.cs

```diff
@@ -84,6 +84,17 @@ public static InternalServerException OfShareableException(Exception exception,
         );
     }
 
+    public static InternalServerException OfRequestTimeoutException(Exception exception, string traceId)
+    {
+        var message = $"Request timed out. If reporting this issue, please include TraceId={traceId}";
```

Comment:
IIRC this "please copy trace id when reporting" message is already part of the overall response? Bit pointless to define such developer-oriented message over here.

I'd also go a little bit more into the reasoning why request has timed out, i.e. make it clear that this is our internal "max request processing duration" timeout or something.

Reply:
well, that's not part of log message. That's user facing message that's returned from API response.